src/sentry/runner/commands/cleanup.py

```diff
@@ -147,77 +146,76 @@ def is_filtered(model):
                     project=project or '*',
                 )
             )
+
+        # TODO: filtering won't take into account "app-level" cascades done in deletions code...
+        #       e.g. filtering out GroupTagValue won't work when we delete a Group here, because all
+        #            child relations are handled within the deletions code.
```

Comment:
Not sure if we care or not? If we do, it means I'd have to push down filtering of models to the deletion code.

Reply:
I don't think this is worth considering. In theory, it'd already be deleted, or will be deleted real soon anyways.